Day-I-Social-Card/social-card/src/components/CardComponents/CardBanner.js

```diff
@@ -1,2 +1,10 @@
 import React from 'react';
 import './Card.css';
+
+const CardBanner = () => (
+  <div className="banner-container">
```

Comment:
Since you are only returning one HTML element anyway, you don't need the div tag parent.